packages/terra-application-header/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra App Delegate
```

Comment:
This file is for terra app delegate but is listed as the readme for terra-application-header? 😕